toolkits/landcover/impl/composites.js

```diff
@@ -95,11 +94,15 @@ function createMedioidComposite(collection, indexBand) {
   indexBand = indexBand === undefined ? 0 : indexBand;
   var median = collection.select(indexBand).median();
 
-  var mosaic = collection.map(function(img) {
+  // Add a band containing the difference between the index band value for each
+  // pixel and the median of the index band across all pixels in the collection.
+  var indexedCollection = collection.map(function(img) {
```

Comment:
Because we're built for method chaining, if we can elide variables only used once without losing meaning, it's better to do so.  There's no reason to hold the intermediate collection; all we're going to do to it is mosaic it.

Reply:
Ack. I'll make it so.